### PR TITLE
feature - view forgot password screen

### PR DESCRIPTION
**Pull request by at-cuongdoan**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/ForgotPassword/ForgotPasswordViewController.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+//
+//  Created by Cuong Doan M. on 12/1/17.
+//  Copyright © 2017 Cuong Doan M. All rights reserved.
+//
+
+import UIKit
+import MVVM
+
+class ForgotPasswordViewController: UIViewController, MVVM.View {
+    @IBOutlet weak var emailTextField: UITextField!
```
Comment (at-hieutran) [inline MyApp/View/ForgotPassword/ForgotPasswordViewController.swift]: Thêm Mark vô

File: MyApp/View/ForgotPassword/ForgotPasswordViewController.swift [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+//  Tutorial
+//
+//  Created by Cuong Doan M. on 12/1/17.
+//  Copyright © 2017 Cuong Doan M. All rights reserved.
+//
+
+import UIKit
+import MVVM
+
+class ForgotPasswordViewController: UIViewController, MVVM.View {
```
Comment (at-hieutran) [inline MyApp/View/ForgotPassword/ForgotPasswordViewController.swift]: Final

File: MyApp/View/ForgotPassword/ForgotPasswordViewController.swift [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    @IBOutlet weak var emailTextField: UITextField!
+
+    var viewModel = ForgotPasswordViewModel()
+
+    override func viewDidLoad() {
```
Comment (at-hieutran) [inline MyApp/View/ForgotPassword/ForgotPasswordViewController.swift]: Thêm Mark vô

File: MyApp/View/ForgotPassword/ForgotPasswordViewController.swift [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        let email = emailTextField.text
+        if !(email.isEmpty) && viewModel.isValidEmail(testStr: email) {
+            let alert = UIAlertController(title: kNotification, message: kMessage, preferredStyle: UIAlertControllerStyle.alert)
+            alert.view.tintColor = .black
+            alert.addAction(UIAlertAction(title: kOK, style: .default, handler: { _ in self.navigationController?.popViewController(animated: true) }))
+            present(alert, animated: true, completion: nil)
+        }
+    }
+
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
```
Comment (at-hieutran) [inline MyApp/View/ForgotPassword/ForgotPasswordViewController.swift]: Gom lại dưới ViewDidLoad.
Thêm Mark vô

File: MyApp/View/ForgotPassword/ForgotPasswordViewController.swift [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        super.touchesBegan(touches, with: event)
+        var frame = view.frame
+        frame.origin.y = 0
+        view.frame = frame
+
+        view.endEditing(true)
+    }
+}
+
+extension ForgotPasswordViewController: UITextFieldDelegate {
```
Comment (at-hieutran) [inline MyApp/View/ForgotPassword/ForgotPasswordViewController.swift]: Thêm Mark vô

**Comment by at-hieutran** [comment]
@at-cuongdoan fix conflict e ơi

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Comment by at-hieutran** [comment]
@at-cuongdoan conflict e ơi

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Pull request closed without merging**